### DISCUSSION TITLE
*UPDATED* AutoOrtho installation guide 0.5.1| Drishal MAC2

### DISCUSSION THREAD

**Discussion by DrishalMAC2** [kubilus1/autoortho#360] [Show and tell]
Hey guys, 

I've made an updated install tutorial for the new AutoOrtho version! Hope i helps.

https://www.youtube.com/watch?v=iwojq1G_vPM

**Comment by kubilus1** [comment]
Thanks!  Do you plan to continue to update tutorials from time to time?  I don't have any big changes to the process but 0.6.x does also and a requirement to set the X-Plane install dir as well.  Pretty minor adjustment I think.

**Reply by DrishalMAC2** [reply]
Yep I will make a new tutorial when required! Hoping we can work together moving forward to make the install process as easy as possible for users!

**Comment by Jeff2122** [comment]
@DrishalMAC2 Nice work on the video. I finally got time to watch it today. I didn't even know about it until someone put a link up, it is in a strange place on the Org and needs to be stickied in the AutoOrtho forum. 
I think you hit the major points of the install, and most should be able to get it working just by watching your work. But as you know there are nuances and some will have to be coached as there is always some Windows (computer) knowledge required to nudge things along. I don't mind helping folks at all if I have answers.

**Reply by DrishalMAC2** [reply]
Yep I've had around 50 people contact me with issues. Each of which I have helped diagnose and fix the issue. Sometimes they don't install it properly or sometimes it just needs winFSP instead of dokan. Most of the work is helping people after the video, not creating the video itself haha.

**Comment by Jeff2122** [comment]
I hear you. I have been on another part of the forum helping people too. We should team up. Maybe a Discord channel manned 24/7 is in order...

**Reply by DrishalMAC2** [reply]
Yep for sure. The way I've been doing it is with an XP12 help section in my discord server and that's working well. Maybe you could join and help me out there? I'm planning to make a new AO 0.6 install video so should have an influx of people coming through for help. 

Question: are you involved with the AO project? Github doesn't really say who is involved and who is just using it.

https://discord.gg/e9QHFWcuRP

**Comment by Jeff2122** [comment]
No. I am just an interested party getting over my head trying to help folks get it running. I do have time to play around with it. Not sure what my success rate is though. Probably confusing more. I am taking notes. I have tried and tried to break it but nothing goes wrong unless I do dumb stuff. So it's the dumb stuff I'm trying to capture, if that makes sense. Maybe they're common mistakes. 
Dumb stuff #1. Do NOT install AO into your XP12 root folder. It will uninstall XP12 100% when you use Apps & Features to uninstall.

**Reply by DrishalMAC2** [reply]
I just today uninstalled AO using the windows apps thing and it was fine. When I did that it didn't actually delete any files in custom scenery or my user folder. I then deleted the AO from my user folder and used the installer to install 0.6.1. Didn't have to reinstall my scenery packages at all and it seems to work OK. Maybe that issue is fixed now?

**Reply by DemonTraitor** [reply]
Not sure how AO works for some but not others. I am a developer and cannot get it to work whatever I do, so just use bog standard ortho. never known an app that has such a random success rate of working...

**Reply by kubilus1** [reply]
> Do NOT install AO into your XP12 root folder. It will uninstall XP12 100% when you use Apps & Features to uninstall.

Yeah, if you have AO use the _exact same_ install folder as X-Plane yeah, you don't want to do that.  lol.  The uninstaller will just merrily remove stuff as it will.

**Reply by Jeff2122** [reply]
@DemonTraitor  Yeah, it can be finicky at times. We have to consider that this is a Linux port to Windows and is experimental, hence the version number starting with 0 (zero). I suppose @kubilus1 will remove that experimental wording/versioning once it is stable for all and released as an App for the world to unite with.
Please do us a favor and join us in this merry circus to try new things and make this a successful endeavor.

**Reply by kubilus1** [reply]
There are just many more variations and inconsistencies on Windows systems.  These virtual filesystem features are built into Linux and work great, whereas there isn't anything equivalent on Windows.  Most issues are some gotcha regarding Dokan or WinFSP that you'll see with other projects as well.

**Comment by Jeff2122** [comment]
I believe it is fixed. So with version 3 where I started using AO, things were totally different, manual and tedious. Now the Windows packager is responsible for taking much of the hassle out. And installing the VFS is similar, easy peasy as long as you use the .msi it can't be scrood up.

**Reply by DrishalMAC2** [reply]
Yeh I've been using the .exe and its really nice.

**Comment by Jeff2122** [comment]
I am in your Discord Server. I am not much for Discord - a noob. Haven't used it for ages. My nick is Elkroc. May change it later.

**Comment by Jeff2122** [comment]
@DrishalMAC2  would you please send a link to your Discord to @Semern55555 so he can climb on the help wagon. The discord channel seems to be by invite only.
Note: His nick didn't pop up so if you can't find him I'll get more info.

**Reply by DrishalMAC2** [reply]
Hiya, yep I think I've seen them in the discord. Helping them out now

**Reply by Jeff2122** [reply]
Eggzullent!

**Comment by Jeff2122** [comment]
I was going to see the effect of hitting Quit after the server is up and running well. I changed "Hide" from true to false in the .autoortho file, but the GUI still disappears. Was there another intent for that boolean other than to keep the GUI up?
